### PR TITLE
Create server.js on express

### PR DESCRIPTION
**Pull request by SergeyKovalchuk**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var app = express();
+
+app.set('port', 3000);
+
+app.use(express.static(__dirname + '/build'));
```
Comment (yankouskia) [inline server.js]: why have you added this?

File: server.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+app.get('/', function (req, res) {
+  	res.send('Hello World!');
+});
+
+app.use(function(req, res, next) {
+	if (req.url == '/someUrl') {
```
Comment (yankouskia) [inline server.js]: what is it?

File: server.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		res.use('TestUrl');
+	} else {
+		next();
+	}
+})
+
+app.use(function(req, res, next) {
+	var err = new Error('Page Not Found Sorry!');
```
Comment (yankouskia) [inline server.js]: what is it?

File: server.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+	err.status = 404;
+	next(err);
+});
+
+
+app.use(function(err, req, res, next) {
+	if (app.get('env') === 'development') {
```
Comment (yankouskia) [inline server.js]: at this moment we don't have any environment variables like prod and dev

File: server.js (hunk 1 above)
Comment (Eugenekravch) [inline server.js]: maybe we add config for server settings?(for ex. port number)

**Pull request closed without merging**